src/Sulu/Bundle/AdminBundle/Resources/js/components/Chip/tests/Chip.test.js

```diff
@@ -1,44 +1,44 @@
 // @flow
 import React from 'react';
-import {render, shallow} from 'enzyme';
+import {fireEvent, render, screen} from '@testing-library/react';
 import Chip from '../Chip';
 
 test('Should render chip with children', () => {
-    expect(render(<Chip value={{}}>Name</Chip>)).toMatchSnapshot();
+    const {container} = render(<Chip value={{}}>Name</Chip>);
+    expect(container).toMatchSnapshot();
 });
 
 test('Should render medium, primary chip with children', () => {
-    expect(render(<Chip size="medium" skin="primary" value={{}}>Name</Chip>)).toMatchSnapshot();
-});
-
-test('Should render chip with delete icon', () => {
-    expect(render(<Chip onDelete={jest.fn()} value={{}}>Name</Chip>)).toMatchSnapshot();
+    const {container} = render(<Chip size="medium" skin="primary" value={{}}>Name</Chip>);
+    expect(container).toMatchSnapshot();
 });
```

Comment:
Looks this test was accidently removed?

Reply:
Oh sorry, yes that test wasn't removed intentionally. Thanks for pointing out!